relayer-engine/src/storage/storage.ts

```diff
@@ -123,18 +123,25 @@ export class Storage {
     lastSeenSequence: number,
   ): Promise<void> {
     return this.store.runOpWithRetry(async redis => {
-      this.logger.debug(`setEmitterRecord`);
+      this.logger.debug(`setEmitterRecord`, {
+        chainId: chainId,
+        emitterAddress: emitterAddress,
+        lastSeenSequence: lastSeenSequence,
+      });
       while (true) {
         const record: EmitterRecord = { lastSeenSequence, time: new Date() };
         const key = emitterRecordKey(pluginName, chainId, emitterAddress);
 
         const entry = await this.getEmitterRecordInner(redis, key);
-        this.logger.debug(`Got emitterRecord ${JSON.stringify(entry)}`);
+        this.logger.debug(`Got emitterRecord ${JSON.stringify(entry)}`, {
+          entry: entry,
+        });
```

Comment:
```suggestion
        this.logger.debug(`Got emitterRecord`, {
          entry: entry,
        });
```

Does it make sense to stringify and log `entry`?